### ISSUE TITLE
mlDeleteModuleTimestampsFile fails with missing input or output annotation

### ISSUE THREAD

**Issue by peetkes** [marklogic-community/ml-gradle#617]
I just pulled the latest version of ml-gradle (4.3.4) and tried to run one of the examples with gradle version 7.1.1. Also tried it with 6.9.1 with the same result.

It fails with 
```
> Task :mlDeleteModuleTimestampsFile FAILED
:mlDeleteModuleTimestampsFile (Thread[Execution worker for ':',5,main]) completed. Took 0.003 secs.

FAILURE: Build failed with an exception.

* What went wrong:
A problem was found with the configuration of task ':mlDeleteModuleTimestampsFile' (type 'DeleteModuleTimestampsFileTask').
  - In plugin 'com.marklogic.ml-gradle' type 'com.marklogic.gradle.task.DeleteModuleTimestampsFileTask' property 'filePath' is missing an input or output annotation.

    Reason: A property without annotation isn't considered during up-to-date checking.

    Possible solutions:
      1. Add an input or output annotation.
      2. Mark it as @Internal.

    Please refer to https://docs.gradle.org/7.1.1/userguide/validation_problems.html#missing_annotation for more details about this problem.

```

This can be fixed by changing the code in DeleteModuleTimestampsTask

When I make String filePath a class property and annotate it with @Internal
The plugin works.

**Comment by rjrudin** [comment]
Closed via #619

**Comment by rjrudin** [comment]
Will be released in 4.3.5
